.github/workflows/test.yml

```diff
@@ -331,7 +331,7 @@ jobs:
 
   # Test that Zebra syncs and fully validates a few thousand blocks from a cached post-checkpoint state
   test-stateful-sync:
-    name: Test validation sync from cached state
+    name: Test full validation sync from cached state
```

Comment:
I think @dconnolly changed this because it was confusing calling it `full`. Maybe changing it in Mergify would be better (?)

Reply:
This is the simplest fix that gets CI merging PRs again.

I'm happy to discuss this on a follow-up PR that changes the name universally.

Reply:
Yeah I may have, it's fine I forgot about the requirements on the human-readable names, not just the workflow file names or id's